### PR TITLE
Russian Language Added

### PR DESCRIPTION
**Pull request by SourangshuGhosh**

### CODE REVIEW

**Comment by SourangshuGhosh** [comment]
@rezendi @technicalpickles  I have added Russian translation of the readme file in this repository. Can you please review these Pull Request?

File: Guide_ru.md [hunk 1]
```diff
@@ -0,0 +1,304 @@
...
+
+Мы ожидаем, что вам не нужны наши определения программного обеспечения, компьютера и других терминов. Мы представляем, что у вас есть ваши собственные компьютеры, вероятно, намного более продвинутые, чем у нас, и, возможно, принципиально иным образом спроектированные. Как только вы поймете обзор и руководство ниже, вы легко сможете получить доступ ко всем данным.
+
+Тем не менее, возможно, что у вас есть компьютеры ниже наших, или даже нет компьютеров вообще. В случае такой возможности мы подготовили несжатый, некодированный, понятный человеку рулон данных, который мы называем Дерева технологий. Дерево технологий содержит информацию о наших фундаментальных технологиях, наших компьютерах и нашем программном обеспечении, в надежде, что со временем вы сможете Используйте эти знания для воссоздания компьютеров, которые могут использовать программное обеспечение с открытым исходным кодом в этом архиве.
```
Comment (fo2rist) [inline Guide_ru.md:29]: Hey! Some of the word in this file do not look like a correct translation into Russian (they're more like one-to-one mappings). I suggest some native speaker to review this whole file to make sure the original meaning is preserved.

E.g. in this line 
```suggestion
Тем не менее, возможно, что у вас есть компьютеры хуже наших, или даже нет компьютеров вообще. В случае такой возможности мы подготовили несжатый, некодированный, понятный человеку рулон данных, который мы называем Дерева технологий. Дерево технологий содержит информацию о наших фундаментальных технологиях, наших компьютерах и нашем программном обеспечении, в надежде, что со временем вы сможете Используйте эти знания для воссоздания компьютеров, которые могут использовать программное обеспечение с открытым исходным кодом в этом архиве.
```
the word `ниже` means `lower [height]`, while it should mean `worse`. That's not the only example, just a demonstration of the potential issue.

**Comment by rezendi** [comment]
Thanks for the PR. We very much appreciate your work, but at least at the moment, we're limiting the translations of the Guide to professional translations. We may well revisit this in future and move to a more expansive list of crowdsourced translations, however, in which case this PR will be reopened and reexamined.

**Comment by rezendi** [comment]
Reopening to add to community guides.

**Pull request merged**